pages/common/mysqldump.md

```diff
@@ -11,3 +11,7 @@
 - Backup all databases redirecting the output to a file (user will be prompted for a password):
 
 `mysqldump --user {{user}} --password --all-databases > {{path/to/file.sql}}`
+
+- Backup all databases from a remote host and redirecting the output to a file (user will be prompted for a password):
```

Comment:
Perhaps this will read better?
```suggestion
- Backup all databases from a remote host, redirecting the output to a file (user will be prompted for a password):
```